MAINTAINER_GUIDE.md

```diff
@@ -0,0 +1,37 @@
+# Maintainer's Guide
```

Comment:
Yeah, I'm aware of that file, but I don't this is what it is. Contributing guidelines is something different – conventions, tools, style, etc. This is more for advanced contributors, maintainer-level, just so that they could understand the decisions. Most contributors won't need this

Reply:
I believe we should have a `CONTRIBUTING.md` anyways, whether this or another file. I agree, this is not your typical `CONTRIBUTING.md`, but it is also useful and should be linked it somewhere (whether in `CONTRIBUTING`, `README`, or elsewhere)

Reply:
Sure, we'll add it at some point and make this file discoverable. But for now it works as it is